src/examples/src/widgets/context-menu/Basic.tsx

```diff
@@ -11,11 +13,16 @@ export default factory(function Basic({ middleware: { icache } }) {
 		'This is some text that has a context menu. Right click to view the context menu and take an action on selected text'
 	);
 	const printedText = icache.getOrSet('printedText', '');
+	const memoryTemplate = createMemoryTemplate();
 
 	return (
 		<virtual>
 			<ContextMenu
-				options={options}
+				resource={{
```

Comment:
I'm a bit out of loop on the resource stuff, but it feels like a bit complex to have to create a template to create a resource just to pass an array of data.

Reply:
yes, agreed. We will be working on providing suitable helpers for these to make it a lot easier and simpler.

Reply:
have made these simpler